packages/core/adder/execute.ts

```diff
@@ -74,7 +74,7 @@ export async function executeAdders<Args extends OptionDefinition>(
 
     let workingDirectory: string | null;
     if (isTesting) workingDirectory = remoteControlOptions.workingDirectory;
-    else workingDirectory = await determineWorkingDirectory(commonCliOptions.path);
+    else workingDirectory = determineWorkingDirectory(commonCliOptions.path);
```

Comment:
Just out of interest: Is this something a tool highlighted to you or something you found out by accident?

Coming mainly from C#, my IDE and even the compiler would have yelled at me for this.

Reply:
Typescript's LSP should provide a warning about it. Here's what it looks like in vscode, although it's quite subtle:
![img](https://i.imgur.com/j89CBuG.png)

---

We could add the [`@typescript-eslint/eslint-plugin`](https://typescript-eslint.io/) plugin with it's recommended typed defaults (which are pretty solid, IMHO) which has a rule to catch these particular cases:
![img](https://i.imgur.com/NlfH1vu.png)

Reply:
That's interesting. I have never seen something like this in vscode. Good catch, and good suggestion. I will add that eslint plugin later.